### ISSUE TITLE
Add az command to access Azure ResourceHealth status

### ISSUE THREAD

**Issue by jpbuecken** [Azure/azure-cli#28055]
**Related command**

**Is your feature request related to a problem? Please describe.**

Azure announces maintenances or defects of a service (e.g virtual machine) in the Azure ResourceHeath API endpoint.

Currently, the az cli does not have a command to access this api endpoint and read announcements via cli.

**Describe the solution you'd like**

Please implement a method to access the api endpoint. Basically, this is the method with the az rest implementation:
**Describe alternatives you've considered**

```
az rest --method get --url https://management.azure.com/subscriptions/${ARM_SUBSCRIPTION_ID}/providers/Microsoft.ResourceHealth/availabilityStatuses
```
**Additional context**

**Comment by yonzhan** [comment]
Thank you for opening this issue, we will look into it.

**Comment by yanzhudd** [comment]
Hi @jpbuecken,
sorry that the info provided in this feature request is not sufficient for us to develop, please use [feature request template](https://github.com/Azure/azure-cli/issues/new?assignees=&labels=feature-request&projects=&template=Service_team_request.yml) to re-raise a feature request.

**Comment by jpbuecken** [comment]
@yanzhudd It looks like the formular you linked is something internal for Microsoft.
There are mandatory fields that only a developer from MS can fill in.
Example:
How should I answer this:

    No need to upgrade Python SDK or the Python SDK is read

or

a Target Date, someone from product manager ID from Micrsosoft and a developer ID from Microsoft. 

But I'm happy if I can define the target date. Can you do it tomorrow ;-)

Long story short. It looks like the document is not for community. If you need further information, please ask what you are missing.

**Comment by jpbuecken** [comment]
@yanzhudd @yonzhan : 

> Long story short. It looks like the document is not for community. If you need further information, please ask, and tell me what you are missing.

Any news?
